### DISCUSSION TITLE
Cloudflare OSS Sponsorships program

### DISCUSSION THREAD

**Discussion by hatemhosny** [okikio/bundle#52] [General]
@okikio 

Cloudflare has launched their Open Source Software Sponsorships program:
https://blog.cloudflare.com/cloudflare-new-oss-sponsorships-program/

They already sponsor projects like unpkg, cdnjs and jsdelivr among others.

I thought that can be useful for your project, specifically for the API

I hope that helps

**Comment by okikio** [comment]
@hatemhosny Yeah, I'll take a look, thanks
